RELEASING.md

```diff
@@ -37,61 +48,86 @@ Label PRs with the next major version label (e.g. `3.0.0`) and merge changes int
 
 * OpenSearch: `main` = 3.0.0, `2.x` = 2.10.0 `1.x` = 1.4.0, and `1.2` = 1.2.5
 * job-scheduler: `main` = 3.0.0.0, `2.x` = 2.10.0.0 , `1.x` = 1.4.0.0 and `1.2` = 1.2.5.0
 
-## Tagging
+## Releases
 
-Create tags after a release that match the version number, `major.minor.patch`, without a `v` prefix.
+### Major Releases
 
-* [OpenSearch tags](https://github.com/opensearch-project/OpenSearch/tags): [1.0.0](https://github.com/opensearch-project/OpenSearch/releases/tag/1.0.0)
-* [job-scheduler tags](https://github.com/opensearch-project/job-scheduler/tags): [1.0.0.0](https://github.com/opensearch-project/job-scheduler/releases/tag/1.0.0.0)
+Major releases contain breaking changes, new features, and bug fixes. All artifacts released in this organization only do major releases when there are significant breaking changes ready for release that warrant a major upgrade. To avoid major upgrades we strive to build features in a backwards compatible way as much as possible, therefore it's common for a distribution to go multiple years without a major release.
 
-For a discussion on whether to add a prefixing `v` to release tags, see [#35](https://github.com/opensearch-project/.github/issues/35).  
+OpenSearch releases new major versions only when there are a critical mass of breaking changes (e.g. changes that are incompatible with existing APIs). These tend to be tied to Lucene major version releases, and will be announced in the forums at least 4 weeks prior to the release date. Once we become aware of the need for a major version, we will start a major version release window which will be similar to a minor release, except for two things: 1) participation is mandatory for all components, and 2) the release window will be at least 4 weeks long to accommodate the testing required. 
 
-## Release Labels
+### Minor Releases
 
-Repositories create consistent release labels, such as `v2.9.0`, `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
+Minor releases for large distributions such as OpenSearch are scheduled on regular intervals, contain new features and bug fixes, and must not contain any breaking changes.
 
-## Release Process
+### Patch Releases
 
-OpenSearch follows semver, which means we will only release breaking changes in major versions. All minor versions are compatible with every other minor version for that major. For example, 1.2.0 will work with 1.3.2, 1.4.1, etc, but may not work with 2.0.
+Patch releases are reserved for high-severity CVEs, critical bug fixes (e.g. causes users to rollback after an upgrade), or significant regressions in non-experimental features (e.g. feature unusable), and must not contain any new features.
```

Comment:
Critical bugs should be limited to existing features (not newly introduced in the current minor release), where users who are already utilizing the feature must rollback due to issues such as broken functionality, significant performance degradation, or unexpected failures rendering the feature unusable.

Reply:
It's important to emphasize existing features (not newly introduced) so that feature/component owners understand that incomplete or poorly developed features should not rely on the possibility of a patch release for completion. Also, with new feature we are not breaking user expectation, so it doesn't demand patch release either.

Reply:
Can you please suggest some text that I can copy-paste?

Reply:
how about this

```suggestion
Patch releases are reserved for addressing high-severity CVEs, critical bug fixes (e.g., issues causing users to roll back after an upgrade), or significant regressions in non-experimental features (e.g., feature rendered unusable). These releases must not introduce any new features.

Whether a fix qualifies as a critical bug should be proposed by the PR contributor or issue creator and triaged by the maintainer, with input from the community. The severity and impact of a bug in defining a critical issue can vary based on the type of repository. For example, a UX bug might be critical for end-user customers and could prevent them from using a feature, requiring domain expertise to assess. In contrast, backend repository bugs are generally more straightforward to triage.
```

Reply:
Taking first paragraph.

For the second I shortened a bit

_Whether a fix qualifies as a critical bug should be proposed by the PR contributor or issue creator, and triaged by the maintainers of the repo with input from the community. The severity and impact of a bug in defining criticality of an issue can vary. For example, back-end APIs are fairly straightforward to triage, while a UX bug may prevent users from using a feature and may require some domain expertise to assess._